spicelib/editor/spice_editor.py

```diff
@@ -681,6 +681,8 @@ class SpiceEditor(SpiceCircuit):
     def __init__(self, netlist_file: Union[str, Path], encoding='autodetect', create_blank=False):
         super().__init__()
         self.netlist_file = Path(netlist_file)
+        if self.circuit_file.suffix != '.net':
+            _logger.warning(f"Netlist file with wrong suffix {self.netlist_file.suffix}")
```

Comment:
I suggest changing it to:
`if self.circuit_file.suffix not in ['.net', '.cir']:`

You mentioned that different spice editor has a different suffix. I agree not to limit that.
However, I want to remind you that some pieces of the code may force a '.cir' netlist to '.net'. (In this circumstance, running simulator will fail). This might need optimization.